### PR TITLE
Fixed customization for ADLS gen 2 storage types

### PR DESCRIPTION
**Pull request by wawon-msft**
This PR contains a fix to customization to support a storage account type that was added earlier. No Swagger spec changes had been made.

---

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/HDInsight/Management.HDInsight/Customizations/Models/AzureStorageInfo.cs [hunk 1]
```diff
@@ -57,13 +57,5 @@ public AzureStorageInfo(string storageAccountName, string storageAccountKey, str
             this._storageAccountKey = storageAccountKey;
             this._storageContainer = storageContainer;
         }
-
-        public string StorageAccountUri
```
Comment (wawon-msft) [inline src/SDKs/HDInsight/Management.HDInsight/Customizations/Models/AzureStorageInfo.cs]: This property isn't used anywhere, and is likely an artifact of legacy code.

Comment (dsgouda) [reply src/SDKs/HDInsight/Management.HDInsight/Customizations/Models/AzureStorageInfo.cs]: This is a breaking change and needs the package version to be set to `3.1.0-preview`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from versioning

File: src/SDKs/HDInsight/Management.HDInsight/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -21,8 +21,8 @@
 
 [assembly: AssemblyTitle("Microsoft Azure HDInsight Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure HDInsight.")]
-[assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.0.3.0")]
+[assembly: AssemblyVersion("3.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/HDInsight/Management.HDInsight/Properties/AssemblyInfo.cs]: The `AssemblyVersion` must remain unchanged for non major version bumps

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from the minor comment

**Review by dsgouda — APPROVED** [review]
LGTM subject to CIs passing

**Comment by wawon-msft** [comment]
Thanks!

**Pull request merged**